zerver/actions/realm_settings.py

```diff
@@ -800,3 +810,65 @@ def do_send_realm_reactivation_email(realm: Realm, *, acting_user: Optional[User
         language=language,
         context=context,
     )
+
+
+def do_send_realm_deactivation_email(realm: Realm, acting_user: Optional[UserProfile]) -> None:
+    shared_context: Dict[str, Any] = {
+        "realm_name": realm.name,
+    }
+    deactivation_time = timezone_now()
+    owners = set(realm.get_human_owner_users())
+    anonymous_deactivation = False
+
+    # The realm was deactivated via the deactivate_realm management command.
+    if acting_user is None:
+        anonymous_deactivation = True
+
+    # This realm was deactivated from the support panel; we do not share the
+    # deactivating user's information in this case.
+    if acting_user is not None and acting_user not in owners:
+        anonymous_deactivation = True
+
+    for owner in owners:
+        owner_tz = owner.timezone
+        if owner_tz == "":
+            owner_tz = timezone_get_current_timezone_name()
+        local_date = deactivation_time.astimezone(
+            zoneinfo.ZoneInfo(canonicalize_timezone(owner_tz))
+        ).date()
+
+        if anonymous_deactivation:
+            context = dict(
+                acting_user=False,
+                initiated_deactivation=False,
+                event_date=local_date,
+                **shared_context,
+            )
+        else:
+            assert acting_user is not None
+            if owner == acting_user:
+                context = dict(
+                    acting_user=True,
+                    initiated_deactivation=True,
+                    event_date=local_date,
+                    **shared_context,
+                )
+            else:
+                context = dict(
+                    acting_user=True,
+                    initiated_deactivation=False,
+                    deactivating_owner=acting_user.full_name,
+                    event_date=local_date,
+                    **shared_context,
+                )
+
+        send_email(
+            "zerver/emails/realm_deactivated",
+            to_emails=[owner.delivery_email],
+            from_name=FromAddress.security_email_from_name(language=owner.default_language),
+            from_address=FromAddress.SUPPORT,
+            reply_to_email=FromAddress.SUPPORT,
```

Comment:
I don't think there's ever a reason to set `reply_to_email` to be the same as `from_address` since it's basically just an override for `from_address`.